### DISCUSSION TITLE
page.window.open()

### DISCUSSION THREAD

**Discussion by SKD8** [flet-dev/website#362] [Ideas]
Can we have page.window.open() method in flet so the window opens automatically when an event triggers

**Comment by ndonkoHenri** [comment]
What do you mean by "open automatically"?
Please detail the behaviour you mean.

**Reply by SKD8** [reply]
assume i wanna remind a user about a task in app. instead of putting a notification, page.window.open() will open the app window to show what task user set the reminder for.
and i also tried to maximise the app using page.window.maximized = True if the app is minimised running in and  the background, using threading library. but it don't maximized automatically when running int the background
